### PR TITLE
Add Requiem weapons set

### PR DESCRIPTION
**Pull request by secretdataz**
Requiem set is a weapon set in jRO that make you hit bio lab monsters harder.
Additionally, I've implemented two item bonus `bMagicAddRace2` and `bIgnoreMdefRace2Rate` to use in Requiem set scripts.

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
@rathena/rathena-script-db-developers

**Comment by nanakiwurtz** [comment]
Hmm I can check the items but I'm not sure about the src code

**Comment by nanakiwurtz** [comment]
:ok_hand: Item DB looks good, waiting for someone to review the src code..

File: doc/item_bonus.txt [hunk 1]
```diff
@@ -272,6 +273,7 @@ bonus2 bIgnoreDefRaceRate,r,n;  	Disregard n% of the target's DEF if the target
 bonus bIgnoreDefClass,c;		Disregard DEF against enemies of class c
 bonus bIgnoreMDefRace,r;		Disregard MDEF against enemies of race r
 bonus2 bIgnoreMdefRaceRate,r,n; 	Disregard n% of the target's MDEF if the target belongs to race r
+bonus2 bIgnoreMdefRace2Rate,r,n; 	Disregard n% of the target's MDEF if the target belongs to race2 r
```
Comment (cydh) [inline doc/item_bonus.txt]: `mr` instead of `r` I think.

File: src/map/pc.c [hunk 2]
```diff
@@ -3673,6 +3673,16 @@ void pc_bonus2(struct map_session_data *sd,int type,int type2,int val)
 		sd->coma_race[type2] += val;
 		sd->special_state.bonus_coma = 1;
 		break;
+	case SP_MAGIC_ADDRACE2: // bonus2 bMagicAddRace2,r,n;
+		PC_BONUS_CHK_RACE2(type2, SP_MAGIC_ADDRACE2);
+		if(sd->state.lr_flag != 2)
+			sd->magic_addrace2[type2] += val;
+		break;
+	case SP_IGNORE_MDEF_RACE2_RATE: //bonus2 bIgnoreMdefRace2Rate,r,n;
```
Comment (cydh) [inline src/map/pc.c]: `//bonus2 bIgnoreMdefRace2Rate,mr,n;`

File: src/map/pc.c [hunk 3]
```diff
@@ -3673,6 +3673,16 @@ void pc_bonus2(struct map_session_data *sd,int type,int type2,int val)
 		sd->coma_race[type2] += val;
 		sd->special_state.bonus_coma = 1;
 		break;
+	case SP_MAGIC_ADDRACE2: // bonus2 bMagicAddRace2,r,n;
```
Comment (cydh) [inline src/map/pc.c]: `// bonus2 bMagicAddRace2,mr,n;`

**Comment by cydh** [comment]
The source for codes are good.

**Comment by aleos89** [comment]
The only problem I see now is the size of the race2 array. We will need to increase the `MAX_RACE2_MOBS` define so the last few mobs aren't ignored during parsing.

**Comment by aleos89** [comment]
Increased the define in c6ba347.

**Pull request merged**